src/xrpld/app/tx/detail/applySteps.cpp

```diff
@@ -258,6 +258,11 @@ invoke_calculateBaseFee(ReadView const& view, STTx const& tx)
             "ripple::invoke_calculateBaseFee : unknown transaction type");
         return XRPAmount{0};
     }
+    catch (std::exception const& e)
+    {
```

Comment:
If this is how we're solving the issue, should this be covered by unit tests

Reply:
I reverted to the original, returning XRPAmount. I added some unit tests.